docker/README.md

```diff
@@ -73,16 +73,16 @@ In addition, the following legacy tags are available:
 
 ## Build a [Devito] image
 
-To build the images yourself, all you need is to run the standard build command using the provided Dockerfile. The main difference between the CPU and GPU images will be the base image that will be used.
+To build the images yourself, you only need to run the standard build command using the provided Dockerfile. The main difference between the CPU and GPU images will be the base image used.
```

Comment:
I preferred "all you need", maybe "all you need to do is running ..."

what does grammarly think about this one?

Reply:
Grammarly prefers:
```
To build the images yourself, all you need to do is run the standard build command using the provided Dockerfile. The main difference between the CPU and GPU images will be the base image that will be used.


To build the (default) CPU image, run:
```

Reply:
yes